projects/igniteui-angular/src/lib/core/styles/typography/_base.scss

```diff
@@ -156,7 +156,7 @@
             $line-height: rem(28px),
             $text-transform: none,
             $margin-top: rem(28px),
```

Comment:
@desig9stein Shouldn't `margin-top` be reduced when you add the `margin-bottom`?

Reply:
@damyanpetev No, we are gonna leave it as it is.